template/deployments/helm/charts/templates/configmap.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: configmap
+data:
+
```

Comment:
youll have to have the env variables plumbed into here somehow, I would say the values references should be here